Mathlib/Data/Complex/Basic.lean

```diff
@@ -1239,29 +1239,6 @@ protected def strictOrderedCommRing : StrictOrderedCommRing ℂ :=
 
 scoped[ComplexOrder] attribute [instance] Complex.strictOrderedCommRing
 
```

Comment:
This is right before the end of the section now and can be deleted
```suggestion
```

Reply:
I don't agree, without this line `open scoped ComplexOrder` in other files does not make `Complex.strictOrderedCommRing` an instance.

Reply:
oops, my bad, I read this as a `local` attribute setting, please disregard my comment instead of a `scoped` one.